ddtrace/contrib/kombu/patch.py

```diff
@@ -45,9 +46,13 @@ def patch():
     # *  extracts/normalizes things like exchange
     _w('kombu', 'Producer._publish', traced_publish)
     _w('kombu', 'Consumer.receive', traced_receive)
+
+    # We do not provide a service for producer spans since they represent
+    # external service calls to another service.
+    # Instead the service should be inherited from the parent.
     Pin(
-        service=config.kombu['service_name'],
-        app='kombu'
+        service=None,
```

Comment:
Note that this is a breaking change for kombu, since consumer spans will now inherit the service from the parent instead of being defined by DD_KOMBU_SERVICE_NAME or `DEFAULT_SERVICE`.

@brettlangdon are we okay with this?

We _could_ do something like this for backwards compatibility... but then consumer spans will _never_ inherit from the parent since the default `DEFAULT_SERVICE` is provided 😕.  

```suggestion
        service=get_env("kombu", "service_name", default=DEFAULT_SERVICE), 
```

Reply:
I think if we do it where it only inherits **if** `DD_SERVICE`/global service is defined, then the "breaking change" is ok, since it is opt in.